src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpReplIdeFeatures.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.Extensions.Interactive;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpReplIdeFeatures : AbstractInteractiveWindowTest
+    {
+        public CSharpReplIdeFeatures(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory)
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+        }
+
+        public new void Dispose()
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+            Reset();
+            base.Dispose();
+        }
+
+        // https://github.com/dotnet/roslyn/issues/801
+        // [Fact]
+        public void VerifyDefaultUsingStatements()
+        {
+            SubmitText("Console.WriteLine(42);");
+            this.WaitForLastReplOutput("42");
+        }
+
+        // https://github.com/dotnet/roslyn/issues/801
+        // [Fact]
+        public void VerifyCodeActionsAvailableInCurrentSubmission()
+        {
+            InsertCode("Directory.Exists(\"foo\");");
+            //      <VerifyCodeActions ApplyFix = "using System.IO;" >
```

Comment:
I thought we had disabled code actions in interactive, which might be why this test was commented out. You'll probably have to spelunk through source history to figure out what happened.

In any case my bigger confusion was "commented out random XML with no explanation of where it came from" will definitely get a "hmm?" from me.